src/components/interaction-designer/resource-editors/PerModeResourceEditor.vue

```diff
@@ -28,18 +28,19 @@
             </header>
           </div>
           <div class="ml-auto">
-            <button
-              :aria-controls="`collapse-lang-panel-${block.uuid}-${item.mode}`"
-              :data-target="`#collapse-lang-panel-${block.uuid}-${item.mode}`"
-              aria-expanded="false"
-              class="btn btn-sm btn-primary"
-              data-toggle="collapse"
-              type="button">
-              show/hide
-            </button>
+            <font-awesome-icon
+              :icon="[
+                'fas',
+                isPanelExpanded[`${block.uuid}-${item.mode}`] === true ? 'angle-up' : 'angle-down'
+                ]"
+              class="cursor-pointer text-primary"
+              @click="updateIsPanelExpanded(`${block.uuid}-${item.mode}`)"/>
           </div>
         </div>
-        <div :id="`collapse-lang-panel-${block.uuid}-${item.mode}`" class="resource-panel-body p-2 collapse multi-collapse">
+        <div
+            v-if="isPanelExpanded[`${block.uuid}-${item.mode}`] === true"
```

Comment:
@bulatgab , @object-required: This reminds me about the advantage of `v-if` (compared to `v-show` or any `show` approach)
- with `v-if`: not everything is dispayed, we only render the desired panel
- before, with `collapse show` only, each channel has that component `PerModeResourceEditorRow` (meaning: if wehave `M` blocks and `N` channels, that component is rendered `M x N` times.

[current render]
<img width="1535" alt="Screen Shot 2022-09-30 at 3 53 51 PM" src="https://user-images.githubusercontent.com/68745918/193361801-7af0078e-6d49-489d-9502-e8e8d5877f7b.png">